website/docs/r/iot_role_alias.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "aws"
+page_title: "AWS: aws_iot_role_alias"
+sidebar_current: "docs-aws-resource-iot-role_alias"
+description: |-
+  Provides an IoT role alias.
+---
+
+# aws_iot_role_alias
+
+Provides an IoT role alias.
+
+## Example Usage
+
+```hcl
+resource "aws_iam_role" "role" {
+  name   = "dynamodb-access-role"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {"Service": "credentials.iot.amazonaws.com"},
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iot_role_alias" "alias" {
+  alias    = "Thermostat-dynamodb-access-role-alias"
+  role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/dynamodb-access-role"
```

Comment:
Instead of manually constructing this ARN, we should use the available `arn` attribute:

```suggestion
  role_arn = "${aws_iam_role.role.arn}"
```